JavaScript/JavaScriptSDK/Sender.ts

```diff
@@ -420,9 +416,9 @@ module Microsoft.ApplicationInsights {
             var errors = results.errors.reverse();
             for (var error of errors) {
                 var extracted = payload.splice(error.index, 1)[0];
-                if (error.statusCode == 408 // Timeout
+                if (error.statusCode == 402 // Too many requests over extended time.
+                    || error.statusCode == 408 // Timeout
                     || error.statusCode == 429 // Too many requests.
-                    || error.statusCode == 439 // Too many requests over extended time.
                     || error.statusCode == 500 // Internal server error.
```

Comment:
I think 439 can still happen in their new SDK but it's just not retryable; however, it sounded like they expected as to dump our cache of pending items when we receive a 439 now.